openpower/patches/blackbird-patches/skiboot/0001-platform-add-Raptor-Blackbird-support.patch

```diff
@@ -92,6 +90,15 @@ index 00000000..8817b8c5
 +	return true;
 +}
 +
++static void blackbird_exit(void)
++{
++	/* Let the IPL Observer know that we're done */
```

Comment:
I thought tpearson was less-than-impressed by having this in skiboot? Could we do this using an init script in petitboot userspace?